fbsamples/graph-api-webhooks-samples
# Modernize sample code

Removed parse sample code since the service no longer exists and the code is more or less the same as heroku. 

Testing on heroku was rather cumbersome as you had to look at logs, so also changed the page to simply display all the received webhooks update on page load rather than "it works!"

Test Plan:
1. Deploying code to: https://sample-webhooks-fork.herokuapp.com/
2. Setup app secret in heroku config vars
2. Setup a facebook webhooks subscription with callback URI: https://sample-webhooks-fork.herokuapp.com/facebook from https://developers.facebook.com/apps/{app-id}/webhooks/ with verify_url as "token"
4. Sent a few webhooks samples and confirm they show up, e.g:

``
[
  {
    "entry": [
      {
        "time": 1492726894,
        "id": "1446951058",
        "changed_fields": [
          "about"
        ],
        "uid": "1446951058"
      }
    ],
    "object": "user"
  },
  {
    "entry": [
      {
        "time": 1492726786,
        "id": "1446951058",
        "changed_fields": [
          "about"
        ],
        "uid": "1446951058"
      }
    ],
    "object": "user"
  }
]
``
